frontend/src/modules/application/components/ApplicationForm.jsx

```diff
@@ -1,11 +1,19 @@
 import React from "react"
-import { personalInformationFields, currentProgramInformationFields, currentStatusFields, customQuestions } from "./FormFields"
+import { connect } from "react-redux"
+import { updateField } from "../actions"
 
+import { personalInformationFields, 
+    currentProgramInformationFields, 
+    currentStatusFields, 
+    customQuestions } from "./FormFields"
+
+// Default input field component
 const DefaultInput = ({ label, curValue, onChange, required }) => <div>
     <label style={{ width: "20vw" }}>{required ? label + ':' : label}</label>
     <input style={{ width: "30vw" }} value={curValue} onChange={onChange} required={required} />
 </div>
 
+// Textbox input field component
```

Comment:
I think the name of the const is enough to explain this, personally. The comment feels a bit superfluous to me.